lapis/src/main/kotlin/org/genspectrum/lapis/model/SiloQueryModel.kt

```diff
@@ -193,24 +195,41 @@ class SiloQueryModel(
         sequenceFilters: CommonSequenceFilters,
         sequenceType: SequenceType,
         sequenceNames: List<String>,
-    ) = siloClient.sendQuery(
-        SiloQuery(
-            SiloAction.genomicSequence(
-                type = sequenceType,
-                sequenceNames = mapSequenceNames(sequenceNames, sequenceType),
-                orderByFields = mapSequenceOrderByFields(sequenceFilters.orderByFields, sequenceType),
-                limit = sequenceFilters.limit,
-                offset = sequenceFilters.offset,
+        rawFastaHeaderTemplate: String,
+        sequenceSymbolType: SequenceSymbolType,
+    ): SequencesResponse {
+        val fastaHeaderTemplate = fastaHeaderTemplateParser.parseTemplate(
+            template = rawFastaHeaderTemplate,
+            sequenceSymbolType = sequenceSymbolType,
+        )
+        val cleanedSequenceNames = sequenceNames
```

Comment:
It looks like this isn't handled at the moment either but if `sequenceNames` contains a value that is not in the `referenceGenomeSchema` it is just returned without an error - shouldn't we raise an error at this point? At the moment we just pass this on to SILO